### ISSUE TITLE
CREATE_FAILED: A subnet ID or network interface ID is required to carry out the request

### ISSUE THREAD

**Issue by wallali** [webdigi/AWS-VPN-Server-Setup#36]
Hi. Thanks for the excellent blog post and this script. However I am now getting an error creating the instance:

```
The following resource(s) failed to create: [VPNServerInstance]. . Rollback requested by user.
--
| 18:01:07 UTC+0100 | CREATE_FAILED | AWS::EC2::Instance | VPNServerInstance | The specified instance type can only be used in a VPC. A subnet ID or network interface ID is required to carry out the request.

```

It seems that 

> You must launch your T2 instances into a virtual private cloud (VPC); they are not supported on the EC2-Classic platform

 as per http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/t2-instances.html

**Comment by webdigi** [comment]
it looks like you have deleted your default vpc. do you have another region to test? or request the default vpc to be restored.

**Comment by wallali** [comment]
I did have a VPC in that (N-Virginia) region:

vpc-5e870d3b | available | 172.30.0.0/16 |   | dopt-57bfa635 | rtb-bd70e6d8 | acl-84de57e1

Tried switching to Ohio and it worked without errors this time. Thanks

**Comment by webdigi** [comment]
thanks for your reply. it looks like it is not the default vpc in North Virginia? the system needs a default vpc in the region for it to work.
